microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -92,8 +92,33 @@ Previously, you could not query for the etcd version included with {microshift-s
 //==== Administrator Perspective
 //admin perspectives go here
 
-//[id="microshift-4-15-networking"]
-//=== Networking
+//[id="microshift-4-15-security"]
+//=== Security and compliance
+//
+// This content will be added post-GA, as it is asynchronous content.
+
+[id="microshift-4-15-networking"]
+=== Networking
+
+[id="microshift-4-15-networking-doc-enhancements"]
+==== Networking documentation enhancements
+
+Various networking documentation improvements are now available in the {microshift-short} release.
+
+* *Network features with their customization status.* A new detailed table of the networking features and their customizations that are accessible in a {microshift-short} instance are described in xref:../microshift_networking/microshift-cni.adoc#microshift-nw-customization-matrix_microshift-about-ovn-k-plugin[MicroShift networking customization matrix].
```

Comment:
One small thought/suggestion for your consideration - I think it's totally fine as is now, but do you think this list would be cleaner if you used a colon after the enhancements instead of a period?
```suggestion
* *Network features with their customization status:* A new detailed table of the networking features and their customizations that are accessible in a {microshift-short} instance are described in xref:../microshift_networking/microshift-cni.adoc#microshift-nw-customization-matrix_microshift-about-ovn-k-plugin[MicroShift networking customization matrix].
```

Not sure if it would look better so it's totally up to you, just thought I'd share that suggestion.

Reply:
I also notice that this period is within the bold text and the others are not. Regardless of which punctuation you choose and whether or not to make it bold, I'd just make sure it's consistent throughout the list of enhancements.

Reply:
Ooo nice catch with the bold periods. Yeah we tried using the colons first but we liked the look of the periods better.